src/components/Footer.jsx

```
@@ -2,8 +2,37 @@ import React from "react";
 export default function Footer() {
   return (
-    <footer className="bg-(--color-neutral-dark)">
-      <h1>Footer</h1>
+    <footer className="bg-neutral-900 p-6 text-white flex justify-between items-center relative">
+      <section className="flex flex-col items-start space-y-3">
+        <h1>
+            <img src="../../src/assets/moviefix-logo.png" alt="moviefix" className="h-10" />
+        </h1>
+        <ul className="flex space-x-4">
+          <li>
+            <a href="https://facebook.com" target="_blank" aria-label="Facebook">
```

ESLint

[error] Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendations

(react/jsx-no-target-blank)